cpp/tests/centrality/betweenness_centrality_test.cpp

```diff
@@ -229,4 +305,41 @@ INSTANTIATE_TEST_SUITE_P(
                       BetweennessCentrality_Usecase{500, false, false, true, true, false}),
     ::testing::Values(cugraph::test::Rmat_Usecase(20, 32, 0.57, 0.19, 0.19, 0, false, false))));
 
+INSTANTIATE_TEST_SUITE_P(
+  manhattan_test_pass,
+  Tests_BetweennessCentrality_File,
+  ::testing::Combine(
+    // disable correctness checks for large dataset, use fewer seeds
+    ::testing::Values(BetweennessCentrality_Usecase{50, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{100, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{200, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{std::numeric_limits<size_t>::max(), false, false, false, false, false}),
+    ::testing::Values(cugraph::test::File_Usecase("/home/nfs/howhuang/cugraph/manhattan.csv"))));
+
+INSTANTIATE_TEST_SUITE_P(
+  newyork_test_pass,
+  Tests_BetweennessCentrality_File,
+  ::testing::Combine(
+    // disable correctness checks for large dataset, use fewer seeds
+    ::testing::Values(BetweennessCentrality_Usecase{50, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{100, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{200, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{500, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{1000, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{std::numeric_limits<size_t>::max(), false, false, false, false, false}),
+    ::testing::Values(cugraph::test::File_Usecase("/home/nfs/howhuang/cugraph/newyork.csv"))));
+
+INSTANTIATE_TEST_SUITE_P(
+  california_test_pass,
+  Tests_BetweennessCentrality_File,
+  ::testing::Combine(
+    // disable correctness checks for large dataset, use fewer seeds
+    ::testing::Values(BetweennessCentrality_Usecase{50, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{100, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{200, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{500, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{1000, false, false, false, false, false},
+                      BetweennessCentrality_Usecase{std::numeric_limits<size_t>::max(), false, false, false, false, false}),
+    ::testing::Values(cugraph::test::File_Usecase("/home/nfs/howhuang/cugraph/roadNet-CA.csv"))));
```

Comment:
You don't need to create new test suits to test a new file.

See

https://github.com/rapidsai/cugraph/blob/branch-25.08/cpp/tests/link_analysis/pagerank_test.cpp#L390